### PR TITLE
Part 1

### PR DESCRIPTION
**Pull request by Jocker34**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    "@storybook/addon-interactions": "^6.5.12",
+    "@storybook/addon-links": "^6.5.12",
+    "@storybook/builder-webpack5": "^6.5.12",
+    "@storybook/manager-webpack5": "^6.5.12",
+    "@storybook/node-logger": "^6.5.12",
+    "@storybook/preset-create-react-app": "^4.1.2",
+    "@storybook/react": "^6.5.12",
+    "@storybook/testing-library": "^0.0.13",
+    "babel-plugin-named-exports-order": "^0.0.2",
+    "eslint": "^8.24.0",
```
Comment (alexander-lozovsky) [inline package.json]: Please keep attention on this 
image

CRA already installs eslint, you just need to configure it to run prettier https://github.com/prettier/eslint-plugin-prettier

please remove all unused eslint dependencies, and .eslintrc file as you have eslintConfig already in package.json

File: package.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react": "^7.31.8",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "husky": "^8.0.1",
+    "prettier": "^2.7.1",
+    "prop-types": "^15.8.1",
+    "webpack": "^5.74.0"
```
Comment (alexander-lozovsky) [inline package.json]: why do you need webpack here?

**Review by alexander-lozovsky — CHANGES_REQUESTED** [review]

**Comment by EduardTrutsyk** [comment]
could you please remove all unused/fake/test files and components to make it clean?

File: package.json (hunk 1 above)
Comment (Jocker34) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (Jocker34) [reply package.json]: deleted

File: package.json [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    "@storybook/addon-essentials": "^6.5.12",
+    "@storybook/addon-interactions": "^6.5.12",
+    "@storybook/addon-links": "^6.5.12",
+    "@storybook/builder-webpack5": "^6.5.12",
+    "@storybook/manager-webpack5": "^6.5.12",
+    "@storybook/node-logger": "^6.5.12",
+    "@storybook/preset-create-react-app": "^4.1.2",
+    "@storybook/react": "^6.5.12",
+    "@storybook/testing-library": "^0.0.13",
+    "babel-plugin-named-exports-order": "^0.0.2",
```
Comment (alexander-lozovsky) [inline package.json]: are you sure you need this?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-react": "^7.31.8",
+    "eslint-plugin-react-hooks": "^4.6.0",
```
Comment (alexander-lozovsky) [inline package.json:93]: from all those plugins you configured only 'prettier' and 'react'. So please remove all unused of configure them properly if you really need them

File: src/stories/Button.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```
Comment (RemZiQ) [inline src/stories/Button.jsx]: Hi, @Jocker34. I think we can also remove storybook examples excluding one that you prefer. With this approach, you will achieve more clear future PRs

**Review by EduardTrutsyk — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (Jocker34) [reply package.json]: deleted

File: package.json (hunk 4 above)
Comment (Jocker34) [reply package.json:93]: deleted

File: src/stories/Button.jsx (hunk 5 above)
Comment (Jocker34) [reply src/stories/Button.jsx]: deleted

**Review by alexander-lozovsky — APPROVED** [review]

**Review by RemZiQ — APPROVED** [review]

**Pull request merged**